src/main/java/com/elytradev/architecture/common/proxy/CommonProxy.java

```diff
@@ -53,6 +55,9 @@ public void init(FMLInitializationEvent e) {
     }
 
     public void postInit(FMLPostInitializationEvent e) {
+        if (Loader.isModLoaded("architect")) {
```

Comment:
Is there no better way to do this? I'd much prefer if the mod class is kept unchanged.

Reply:
I use to do this kind of task either in the mod or proxy class; but other suggestions are welcome. It just needs to be loaded after Architect has initialized.